kolibri/plugins/learn/assets/src/views/search-page/index.vue

```diff
@@ -160,7 +153,30 @@
       },
       numFiltered() {
         return this.filteredResults.length;
-      }
+      },
+      noResultsMsg() {
```

Comment:
Addressed in https://github.com/learningequality/kolibri/pull/1514/commits/c0016a7e7e6238a3588d74c1951efd3c818dbd96